### PR TITLE
DDT-991: Requested Equipment Equipment

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 1]
```diff
@@ -467,6 +468,12 @@ CREATE TABLE dcsa_im_v3_0.equipment (
     weight_unit varchar(3) NULL REFERENCES dcsa_im_v3_0.unit_of_measure(unit_of_measure_code)
 );
 
+DROP TABLE IF EXISTS dcsa_im_v3_0.requested_equipment_equipment CASCADE;
+CREATE TABLE dcsa_im_v3_0.equipment_reference (
+    requested_equipment_id uuid NOT NULL REFERENCES dcsa_im_v3_0.requested_equipment(id),
+    equipment_reference varchar(15) NOT NULL REFERENCES dcsa_im_v3_0.equipment(equipment_reference)
+);
+
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:476]: Please move the line below up over this CREATE TABLE... and add an index on equipment_reference

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 2]
```diff
@@ -467,6 +468,12 @@ CREATE TABLE dcsa_im_v3_0.equipment (
...
 -- Supporting FK constraints
 CREATE INDEX ON dcsa_im_v3_0.equipment (iso_equipment_code);
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:478]: Please move this line up over this CREATE TABLE above... and add an index on equipment_reference on the equipment table

Comment (mp-assco) [reply datamodel/initdb.d/03_dcsa_im_v3_0.sql]: Done

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 3]
```diff
@@ -472,6 +473,13 @@ CREATE TABLE dcsa_im_v3_0.equipment (
 
 -- Supporting FK constraints
 CREATE INDEX ON dcsa_im_v3_0.equipment (iso_equipment_code);
+CREATE INDEX ON dcsa_im_v3_0.equipment (equipment_reference);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.requested_equipment_equipment CASCADE;
```
Comment (pimg) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:478]: requested_equipment_equipment is not a table in our DB

**Review by pimg — CHANGES_REQUESTED** [review]

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql (hunk 3 above)
Comment (mp-assco) [reply datamodel/initdb.d/03_dcsa_im_v3_0.sql:478]: Sorry, that was a naming mistake.

**Review by pimg — APPROVED** [review]

**Review by pimg — APPROVED** [review]

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**